hac-node/agent/indexer.go

```
@@ -0,0 +1,861 @@
+	}
+}
+
+func (c *ChainIndexer) settlePR() {
+	c.logger.Info("start settle PR")
+	proposals, err := c.getProposalsByStatus(uint64(hac_types.ProposalStatusProcessing), 0, 100)
+	if err != nil {
+		c.logger.Error("get proposals fail", "err", err)
+	}
+	for _, p := range proposals {
+		if p.ProposerAddress == c.localAddress {
+			_, cnt, err := c.getDiscussionByProposal(p.Id, 0, 1)
```

golangci-lint (1.62.2)

ineffectual assignment to err

(ineffassign)